### PR TITLE
Fix UI modules logs

### PR DESCRIPTION
**Pull request by tbouron**
This does the following:
- enable debug logs for non-production builds
- enable angular runtime logs for non-production builds
- remove the `drop_console` option from webpack to display info, warn and error logs on production builds

### CODE REVIEW

**Comment by tbouron** [comment]
retest this please

**Comment by tbouron** [comment]
FYI, I change the maven version to be 3.5.4 (latest available on the Apache Jenkins nodes as far as I'm aware)

**Comment by tbouron** [comment]
@geomacy Build pass :shipit:

File: ui-modules/catalog/app/index.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ angular.module('app', [ngAnimate, ngCookies, ngResource, brCore, brServerStatus,
 
 
 function applicationConfig($logProvider, $compileProvider) {
-    $logProvider.debugEnabled(IS_PRODUCTION);
```
Comment (ahgittin) [inline ui-modules/catalog/app/index.js]: `!`  :) good fix

**Comment by ahgittin** [comment]
Thanks guys.  Code looks good, and works as advertised.  I can now use `$log` in my code!

Merging.

**Comment by geomacy** [comment]
> FYI, I change the maven version to be 3.5.4 (latest available on the Apache Jenkins nodes as far as I'm aware)

should have thought of that myself...

**Pull request merged**